admin-manual/installation/upgrading.rst

```diff
@@ -179,9 +179,14 @@ First, change the current directory:
 
 Now, run the upgrade-sql task:
 
+.. NOTE::
+
+   Depending on your database size, this task may use a lot of memory so we
+   suggest to run it without memory limit.
```

Comment:
Minor grammatical change here: I would suggest wording it like so:

Depending on your database size this task may use a lot of memory, so we suggest running it without a memory limit.